scripts/generate_abi.py

```diff
@@ -0,0 +1,97 @@
+import json
+import tempfile
+from pathlib import Path
+from typing import Any
+
+import click
+from psutil import Popen
+
+COMPILE_COMMAND = """
+git clone git@github.com:beamer-bridge/beamer.git {beamer_path} &&
+cd {beamer_path} &&
+git fetch &&
+git checkout {commit_id} &&
+$(which ape) compile
```

Comment:
Why do you need to invoke `ape` like that?
It seems you can invoke `git` normally, so why should `ape` be different?

Reply:
i was thinking git would be already there in ci for example and ape is somehow connected to poetry run etc normally that made me do this but i guess there wont be any ape in ci without using some poetry command right?